.gitignore

```diff
@@ -1,7 +1,9 @@
 \[.
 tags
 .vimrc
+.idea
```

Comment:
if you put that into your global .gitignore, you don't have to add it to every project